Report.md

```diff
@@ -1,11 +1,44 @@
-# Sean's Report
+# 31257 ISDM - Assignment 1: Project Report
 
+## Title page 
+## Table of Contents 
+## Acknowledgments 
+## Executive Summary 
+## Body of the Report 
+## References 
+## Appendices
+
+## _Draft area (below)_
 This is an edit to test
 This is also an edit test :D
```

Comment:
Might as well get rid of these while you're at it

Reply:
No